Makefile

```diff
@@ -7,17 +8,26 @@ MONGO_SETTINGS=MONGO_CONNECTION=${MONGO_CONNECTION} \
 	CUSTOMCONNSTR_mongo_collection=${CUSTOMCONNSTR_mongo_collection} \
 	CUSTOMCONNSTR_mongo_settings_collection=${CUSTOMCONNSTR_mongo_settings_collection}
 
-.PHONY: all coverage test travis
+# XXX.bewest: Mocha is an odd process, and since things are being wrapped and
+# transformed, this odd path needs to be used, not the normal wrapper.
+# When ./node_modules/.bin/mocha is used, no coverage information is generated.
+MOCHA=$(shell which mocha ./node_modules/mocha/bin/_mocha | head -n 1)
+ISTANBUL=$(shell which istanbul ./node_modules/.bin/istanbul | head -n 1)
+
+.PHONY: all coverage report test travis
 
 all: test
 
 coverage:
 	NODE_ENV=test ${MONGO_SETTINGS} \
-	istanbul cover ./node_modules/mocha/bin/_mocha -- -vvv -R tap ${TESTS}
+	${ISTANBUL} cover ${MOCHA} -- -R tap ${TESTS}
+
+report:
+	test -f ./coverage/lcov.info && (npm install coveralls && cat ./coverage/lcov.info | ./node_modules/.bin/coveralls) || echo "NO COVERAGE"
```

Comment:
Seems like there should be a line break in here, or the `coverage` & `travis` tasks should lose theirs.